chalicelib_smaht/buckets.py

```diff
@@ -0,0 +1,32 @@
+import boto3
+import json
+from chalicelib_smaht.vars import FOURSIGHT_PREFIX
+from foursight_core.buckets import Buckets as Buckets_from_core
```

Comment:
It would be nice if Core's version of `Buckets` was called something like `CoreBuckets` so that the import here would be clean. Note that Core could still do:

```
class CoreBuckets:
 ... support to be inherited ...

class Buckets(CoreBuckets):   #  class name to be used locally in foursight_core, perhaps for testing, but not inherited
    pass
```